src/PROPOSAL/detail/PROPOSAL/crosssection/CrossSectionDNDX/CrossSectionDNDXInterpolant.cxx

```diff
@@ -62,8 +62,17 @@ size_t CrossSectionDNDXInterpolant::gen_hash(size_t hash) const {
 
 double CrossSectionDNDXInterpolant::Calculate(double energy)
```

Comment:
maybe a private function getting the energy and retransformed v, which both `Calculate` functions call, might prevent duplicated code